docs/api/ref/schemas/tags/tags-write.yaml

```diff
@@ -0,0 +1,27 @@
+type: object
+title: Tags (WRITE)
+description: |
+  Products have list of tags of different types (e.g. categories, labels) that can be replaced or added to with the following properties in API v3.
+properties:
+  categories_tags:
+    type: array
+    description: An array of categories tag entries that will replace existing categories
+    items:
+      $ref: ./input_taxonomy_tag_entry.yaml
+  categories_tags_add:
```

Comment:
Instead of having a _add suffix for all tag properties, we could remove that, and instead have a default parameter like tags_mode which would be "add" or "replace". Similiar to the tags_lc parameter that indicate the default language for tags.